consensus/qbft/src/main/java/org/hyperledger/besu/consensus/qbft/statemachine/QbftRound.java

```diff
@@ -332,7 +332,7 @@ private void peerIsPrepared(final Prepare msg) {
   private void peerIsCommitted(final Commit msg) {
     final boolean wasCommitted = roundState.isCommitted();
     roundState.addCommitMessage(msg);
-    if (wasCommitted != roundState.isCommitted()) {
+    if (!wasCommitted && roundState.isCommitted()) {
```

Comment:
Is this logic identical to the previous logic, but you changed it to be clearer?

Reply:
If so that's fine, just checking

Reply:
This correctly captures false to true transition of committed condition. I saw that blocks were committed without commit quorum before this change. I'll add more details about this change.

Reply:
reverted this change, don't see any issue now . Will fix it in a different PR if needed